diesel_tests/tests/lib.rs

```diff
@@ -2,13 +2,9 @@
 
 #[macro_use]
 extern crate assert_matches;
-extern crate chrono;
+
 #[macro_use]
 extern crate diesel;
-#[cfg(feature = "sqlite")]
-extern crate diesel_migrations;
-extern crate dotenvy;
-extern crate quickcheck;
```

Comment:
Mistake?

Reply:
That's fine. `extern crate …` statements are not required anymore, so we can remove them (even those with a `#[cfg]` on.)